build.psake.ps1

```diff
@@ -124,12 +124,21 @@ Properties {
     $PublishDir     = "$PublishRootDir\$ModuleName"
 
     # The local installation directory for the install task. Defaults to your user PSModulePath.
-    $InstallPath = Join-Path -Path (Split-Path $profile.CurrentUserAllHosts -Parent) -ChildPath "Modules\$ModuleName\$($ModuleDetails.Version.ToString())" 
+    $InstallPath = $null
```

Comment:
Using `$profile` here was causing psake to fail on AppVeyor because it contained `$null`.  Moved the initialization to the variable down to the task so it shows up when the property gets used.